spec.bs

```diff
@@ -654,10 +655,27 @@ To <dfn>generate and score bids</dfn> given an [=auction config=] |auctionConfig
 To <dfn>score and rank a bid</dfn> given an [=auction config=] |auctionConfig|, a [=generated bid=]
 |generatedBid|, a [=leading bid info=] |leadingBidInfo|, a [=string=] |decisionLogicScript|, and a
 [=boolean=] |hasComponentAuction|:
-1. TODO: Assemble and fetch trusted scoring signals
+1. Let |renderUrls| be a new [=list/is empty|empty=] [=list=].
+1. Let |adComponentRenderUrls| be a new [=list/is empty|empty=] [=list=].
+1. If |generatedBid|'s [=generated bid/ad descriptor=] is not null:
+  1. [=list/Append=] |generatedBid|'s [=generated bid/ad descriptor=]'s [=ad descriptor/url=] to
+    |renderUrls|.
+1. If |generatedBid|'s [=generated bid/ad component descriptors=] is not null:
+  1. [=list/Append=] |generatedBid|'s [=generated bid/ad component descriptors=]'s [=ad descriptor/url=]
+    to |adComponentRenderUrls|.
+1. Let |fullSignalsUrl| be the result of [=building trusted scoring signals url=] with |auctionConfig|'s
+  [=auction config/trusted scoring signals url=], |renderUrls|, |adComponentRenderUrls|,
+  |auctionConfig|'s [=auction config/seller experiment group id=].
+
+  Note: Implementations can batch requests by collecting the |renderUrls| and |adComponentRenderUrls| from
+  multiple invokations of [=score and rank a bid=].
+1. Let |trustedScoringSignals| be the result of [=fetching resource=] with |fullSignalsUrl|

```

Comment:
My [PR](https://github.com/WICG/turtledove/pull/532/files#diff-6f5a1d8263b0b0c42e2716ba5750e3652e359532647ac934c1c70086ae3ceddaR806) implements this. It validates the response, but haven't decided whether my way of "parsing the reponse" is enough.
Feel free to review it if you'd like to. Anyway, just want to make sure that we're not duplicating the work.

Reply:
Acknowledged -- I added a TODO. We can point to your new section from here when it lands.